packages/aws-cdk-lib/aws-appconfig/lib/environment.ts

```diff
@@ -154,6 +179,7 @@ export class Environment extends EnvironmentBase {
       public readonly applicationId = applicationId;
       public readonly environmentId = environmentId;
       public readonly environmentArn = environmentArn;
+      public readonly name?: string | undefined;
```

Comment:
```suggestion
      public readonly name?: string;
```
I'm not sure this will be set so probably for environments imported with `fromEnvironmentArn` the `this.name!` in `addDeployment` will throw.
Can you please verify?

Reply:
Yes, it will throw in that scenario and when the name is not provided to `fromEnvironmentAttributes`.  
My understanding is that behavior is the same for the current implementation.